src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -1796,51 +1800,61 @@ private void CheckForSolutionFile()
 				.FirstOrDefault()?.ItemPath;
 		}
 
+		private void GetDesktopIniFileData()
+		{
+			var path = Path.Combine(WorkingDirectory, "desktop.ini");
+			DesktopIni = WindowsIniService.GetData(path);
+		}
+
 		public void CheckForBackgroundImage()
 		{
-			var iniPath = Path.Combine(WorkingDirectory, "desktop.ini");
-			if (!File.Exists(iniPath))
+			var filesAppSection = DesktopIni.FirstOrDefault(x => x.SectionName == "FilesApp");
```

Comment:
How do you add a section? I tried adding `[FilesApp]` but it didn't work.

Reply:
Fixed.